### ISSUE TITLE
clip question

### ISSUE THREAD

**Issue by irislovebest** [yifan123/flow_grpo#203]
Hello, author. Train SD3.5 using pickscore with lora， however it is never clipped. Is this situation normal？ Are other models with this situation, and how about without lora?

**Comment by yifan123** [comment]
If clip_frac = 0, it indicates that the model is updating very slowly. In that case, you can increase the learning rate to improve training efficiency.

**Comment by irislovebest** [comment]
But when I increase the learning rate, the model will crash in a few dozen steps. The effective batchsize is 576, When the learning rate is 3e-4, the reward steadily increases and is not clipped, but when the learning rate is 3e-3, it quickly crashes and is still not clipped

**Comment by MengHao666** [comment]
> But when I increase the learning rate, the model will crash in a few dozen steps. The effective batchsize is 576, When the learning rate is 3e-4, the reward steadily increases and is not clipped, but when the learning rate is 3e-3, it quickly crashes and is still not clipped

similar problem， have u solved it？
